app/views/home/maintainers.md

```diff
@@ -0,0 +1,45 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that identify as a "maintainer" regularly contribute to an open source project.
+
+> What are you asking me to do?
+
+Take one day this week to make your project easier to use and more welcoming to new contributors.
```

Comment:
My first thought when I read _make your project easy to contribute to_ was "that's impossible".

Maybe s/easy/easier/

_"Take one day this week to make your project easier to contribute to"_